google/data_source_google_compute_project.go

```diff
@@ -0,0 +1,54 @@
+package google
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func dataSourceGoogleComputeProject() *schema.Resource {
```

Comment:
looking at how it's structured, we should probably create dataSourceGoogleProject datasource (no Compute in the name) and only expose project number AND project name. GCE is a part of project and may not always be needed to make it a dependency for exposing or project information

google_compute_service_account should be separated into its own data source to better fit into already existing datasources